holidayextras/brands
# UXUI-573 | Adding globals into brands

#### What does this PR do? (please provide any background)
This PR addresses some concerns we've been having recently towards the branding structure thats implemented across HX. We need a global place to store styling that's perhaps not been covered by bootstrap or extra features that we want to introduce that are inherited across the brands. This is not a replacement for Toolkit component styling, it merely plugs the holes missed by bootstrap whilst attending to some of our specific application needs.

#### What tests does this PR have?
None :( 

#### How can this be tested?
Checkout the brands repo locally. NPM link into Pattern Library and double check the global files are compiling in the terminal.

#### What gif best describes how you feel about this work?
![](https://media3.giphy.com/media/BKimimq6MtjAA/giphy.gif) -> Came up for globe search, don't hate me.

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
